src/style.css

```
@@ -0,0 +1,87 @@
+
+/* Fonts */
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 800;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+/* Styles */
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Avenir", serif;
+  font-weight: 300;
+}
+
+.formField {
+  position: relative;
+  margin-top: 20px;
+}
+
+/* Inputs */
+
+.input {
+  font-family: inherit;
+  font-size: 16px;
+  padding: 23px 0 23px 62px;
+  width: 100%;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  outline: none;
+}
+
+.input::placeholder {
+  font-weight: initial;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+.input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.input:focus {
+  font-weight: 800;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+}
+
+.inputSmall {
```

it's better to name classes with kebab-case, not camelCase
```suggestion
.input-small {
```